core/storage/base_model/gae_models_test.py

```diff
@@ -321,7 +321,7 @@ def setUp(self):
         self.model_instance = TestBaseHumanMaintainedModel(id=self.MODEL_ID)
         def mock_put(self):
             # type: (base_models.BaseHumanMaintainedModel) -> None
-            """Function to make changes in datastore entities used for
+            """Function to save the entities to the datastore used for
```

Comment:
Solves [this](https://github.com/oppia/oppia/pull/13413#discussion_r683735283) comment as per earlier PR.

Reply:
This comment is not so clear, it's not clear whether the "used for testing" part applies to the datastore or to the function. Can you rewrite it so that it's unambiguous?

Reply:
Done. PTAL